packages/server/graphql/types/IntegrationProviderMetadataInputOAuth1.ts

```diff
@@ -0,0 +1,28 @@
+import {GraphQLInputObjectType, GraphQLNonNull, GraphQLString} from 'graphql'
+import GraphQLURLType from './GraphQLURLType'
+
+export interface IIntegrationProviderMetadataInputOAuth1 {
+  serverBaseUrl: string
+  consumerKey: string
+  privateKey: string
+}
+export const IntegrationProviderMetadataInputOAuth1 = new GraphQLInputObjectType({
+  name: 'IntegrationProviderMetadataInputOAuth1',
+  description: 'OAuth1 provider metadata',
+  fields: () => ({
+    serverBaseUrl: {
+      type: new GraphQLNonNull(GraphQLURLType),
+      description: 'The base URL used to access the provider'
+    },
+    consumerKey: {
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
+1 if we know the consumerKey or privateKey doesn't have any spaces in it, we can use a GraphQLID type.

Reply:
`privateKey` has line breaks

Reply:
whoa they got rid of that (last time i checked was before the spec existed 😅 ) thanks for correcting my old intel.